crates/ekore/src/harmonics/log_functions.rs

```diff
@@ -126,24 +125,28 @@ pub fn lm15(
     S4: Complex<f64>,
     S5: Complex<f64>,
 ) -> Complex<f64> {
-    -((((pow(S1, 5) + ((10. * pow(S1, 3)) * S2)) + ((20. * pow(S1, 2)) * S3))
-        + ((15. * S1) * (pow(S2, 2) + (2. * S4))))
-        + (4. * (((5. * S2) * S3) + (6. * S5))))
+    -(S1.powu(5)
+        + 10. * S1.powu(3) * S2
+        + 20. * S1.powu(2) * S3
+        + 15. * S1 * S2.powu(2)
+        + 2. * S4
```

Comment:
is there a typo here by chance, i.e. `2. * S4 * S1`? because I guess all weights should add up to 5 and the one possibility you're missing is $S_4 \cdot S_1$

Reply:
yes, good catch! Fixed in 353bba6f